npm-shrinkwrap.json

```diff
@@ -352,8 +352,8 @@
     },
     "fxa-auth-db-mysql": {
       "version": "0.42.0",
-      "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#a09790c20fa8c9f4a2c6f108b86aabf07038220f",
-      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#a09790c20fa8c9f4a2c6f108b86aabf07038220f",
+      "from": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#master",
+      "resolved": "git+https://github.com/mozilla/fxa-auth-db-mysql.git#354fc512d181b002b7aa62740889eaf60511a2d7",
```

Comment:
I'm no export in reading shrinkwrap, but looks like some of these have reverted to checking out master rather than a commit hash?